addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
```

Comment:
@pzuraq actually that's not true. `wait` doesn't work for promises to resolve. It waits for 
1. jQuery's ajax request (not fetch, not raw XHR)
2. The runloop to be totally empty
3. Waiters. Waiters are functions developers or addons can register that can do absolutely anything. A waiter function is pending if it returns a falsy value.

`wait` might not not needed to wait for the event to fire, but we must `return wait()` to allow users to wait for any **asynchronous consequence** of that event.